dashboard/db/migrate/20160510192122_populate_hashed_email_in_users.rb

```diff
@@ -0,0 +1,18 @@
+# A migration that populates (if not already populated) a user's hashed_email
+# for users having an (non-empty) email address.
+
+class PopulateHashedEmailInUsers < ActiveRecord::Migration
+  def change
+    reversible do |direction|
```

Comment:
I think it would be clearer and simpler to have an up method with the logic inside the direction.up block, and a down method that does nothing.